system/modules/core/library/Contao/Model/Registry.php

```diff
@@ -83,16 +89,49 @@ public function count()
 	/**
 	 * Fetch a model by table name and primary key
 	 *
-	 * @param string  $strTable The table name
-	 * @param integer $intPk    The primary key
+	 * @param string  $strTable     The table name
+	 * @param mixed   $varKey       The key
+	 * @param string  $strAlias     An optional alias
```

Comment:
Merely cosmetic but I think we should have a standard here. Usually it is one space after the longest parameter...